### PR TITLE
Include image building for Azure Disk CSI Driver and Operator

### PR DESCRIPTION
**Pull request by huffmanca**
This includes image building for the Azure Disk CSI Driver and Operator.

### CODE REVIEW

File: ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -3,11 +3,23 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.8
```
Comment (bertinatto) [inline ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml:9]: I would expect us to use go 1.16 instead

Comment (bertinatto) [reply ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml:9]: BTW, note that go 1.16 build tools were added recently, so the Dockerfiles should probably be updated as well

Comment (huffmanca) [reply ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml]: This has been updated for both the driver and operator. I've also opened PRs for both of these:

* Driver - https://github.com/openshift/azure-disk-csi-driver/pull/4
* Operator - https://github.com/openshift/azure-disk-csi-driver-operator/pull/6

File: ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml [hunk 2]
```diff
@@ -3,11 +3,23 @@ base_images:
...
     tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.15
```
Comment (bertinatto) [inline ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml:14]: Don't we need to change this too?

Comment (huffmanca) [reply ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml]: I've updated this to `golang-1.16` for the master, 4.8, and 4.9 jobs.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**